core/src/test/resources/sql/mysql-func.iq

```diff
@@ -0,0 +1,33 @@
+# misc.iq - Miscellaneous queries
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+!use mysqlfunc
+!set outputformat mysql
+
+# XML Functions
+
+SELECT ExtractValue('<a>c</a>', '//a');
```

Comment:
Do we need such test in Quidem? Will unit tests are already enough for testing a single function?

Reply:
@amaliujia I suggested adding it to .iq. In my opinion tests in .iq are easier to maintain than the equivalent test in say JdbcTest.java.

@ritesh-kapoor Glad you figured out how to create a test that ran using the mysql function table.

You may need to update the comment at the top of mysql-func.iq.

Reply:
Done :)